halo/app/start.go

```diff
@@ -65,6 +75,34 @@ func (c Config) BackendType() dbm.BackendType {
 	return dbm.BackendType(c.Config.BackendType)
 }
 
+// Register metrics and health endpoints.
+func registerEndpoints(cfg *cmtcfg.Config) {
+	http.Handle("/", promhttp.Handler())
```

Comment:
lets avoid using global http mux

Reply:
Thanks a lot, I didn't know one can specify a local multiplexer! (Why does a global one even exist?) This was the reason why I needed the global variable and `Once.Do`, because test call `Start` within the same process.